openwisp_monitoring/monitoring/migrations/0001_squashed_0023_alert_settings_tolerance_remove_default.py

```diff
@@ -48,9 +48,9 @@ class Migration(migrations.Migration):
 
     dependencies = [
         ('monitoring', '__first__'),
-        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        swapper.dependency(*swapper.split(settings.AUTH_USER_MODEL), latest=True),
         ('contenttypes', '0002_remove_content_type_name'),
-        ('openwisp_notifications', '0003_notification_notification_type'),
+        swapper.dependency('openwisp_notifications', 'Notifications'),
         swapper.dependency('monitoring', 'Metric'),
         swapper.dependency('monitoring', 'AlertSettings'),
```

Comment:
@nemesisdesign aren't these dependencies resolving to the same file? Can we remove them?

Reply:
@pandafy,  should I remove them?

Reply:
Let's wait for @nemesisdesign's opinion before proceeding.

Reply:
Okay! Sure.
Thanks

Reply:
I think we can try to remove them.